src/derivatives/05-diffusion-derivatives.md

```diff
@@ -41,83 +41,81 @@ which has its own unique requirements with respect to:
-    empirical diffusion-weighted data.
+When a model is specified by the data, parameters of the model can be saved
+using the following conventions:
 
-1.  <a name="paramdef-derived">Model-*derived*</a> parameter:
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>]_model-<label1>_param-<param1>_dwimap.nii[.gz]
+            <source_keywords>[_space-<space>]_model-<label1>_param-<param2>_dwimap.nii[.gz]
+            <source_keywords>[_space-<space>]_model-<label1>_dwimap.json
+            <source_keywords>[_space-<space>]_model-<label2>_param-<parama>_dwimap.nii[.gz]
+            <source_keywords>[_space-<space>]_model-<label2>_param-<paramb>_dwimap.nii[.gz]
+            <source_keywords>[_space-<space>]_model-<label2>_dwimap.json
+```
 
-    Value that can be calculated directly from previously estimated
-    *model parameters*, without necessitating reference to
-    the empirical diffusion-weighted data.
+- Files "`<source_keywords>[_space-<space>]_model-<label>_param-<param*>_dwimap.nii[.gz]`"
+    provide model outputs and/or parameters of the model.
 
-For example, consider a diffusion tensor model fit: the number of
-iterations in the optimization algorithm would be an *input* parameter;
-the six unique diffusion tensor coefficients would be the *model fit*
-parameters; the Fractional Anisotropy (FA) would be a *model-derived*
-parameter (as it is calculated from the diffusion tensor coefficients
-rather than the image data).
+- Only one metadata file, "`<source_keywords>[_space-<space>]_model-<label>_dwimap.json`"
+    is allowed for each set of files that has a specific `model-<label>` entity.
 
-### File names
+Example:
 
 ```Text
-<pipeline_name>/
-    sub-<participant_label>/
+my_tensors/
+    sub-01/
         dwi/
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_model.json
-
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.json]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.json]
-```
+            sub-01_model-DTI_param_Sxx_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxy_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxz_dwimap.nii.gz
+            sub-01_model-DTI_param_Syy_dwimap.nii.gz
+            sub-01_model-DTI_param_Syz_dwimap.nii.gz
+            sub-01_model-DTI_param_Szz_dwimap.nii.gz
```

Comment:
If you want for it to be possible / permissible to define the individual components of the tensor model as individual NIfTI images, I think that warrants its own separate discussion. This would in the immediate case lie outside of the specification. The appropriate interpretation of how those files combine together to form a tensor representation would need to be somehow specified, in a way that establishes correspondence between tensor components and file names (unlike #74, where "tensor" is a keyword establishing a mapping from volume index to symmetric rank-2 tensor coefficient). Also the reference orientations could be the image axes rather than scanner axes, in which case "x" / "y" / "z" don't apply.

If the goal here is simply to have an example showing how there can be one JSON file and multiple NIfTIs, that can be achieved without introducing this new problem.

Reply:
I am definitely not attached to the idea that the tensor could be saved in six different files (and doubt anyone really wants to do that). It was indeed the goal to show that one json can refer to multiple niftis, but I agree that this just added more problems, so will remove.